tutorials/networking/high_level_multiplayer.rst

```diff
@@ -388,9 +465,154 @@ have loaded the game scene.
         players.clear()
         server_disconnected.emit()
 
+ .. code-tab:: csharp
+
+    public partial class Lobby : Node
+    {
+        // These signals can be connected to by a UI lobby scene or the game scene.
+        [Signal]
+        public delegate void PlayerConnectedEventHandler(int id, Dictionary<string,string> playerInfo);
+        [Signal]
+        public delegate void PlayerDisconnectedEventHandler();
+        [Signal]
+        public delegate void ServerDisconnectedEventHandler();
+
+        const int Port = 7000;
+        const string DefaultServerIP = "127.0.0.1"; // IPv4 localhost
+        const int MaxConnections = 20;
+
+        // This will contain player info for every player,
+        // with the keys being each player's unique IDs.
+        public Dictionary<long,Dictionary<string,string>> Players = new Dictionary<long,Dictionary<string,string>>();
+        int playersLoaded = 0;
+
+        // This is the local player info. This should be modified locally
+        // before the connection is made. It will be passed to every other peer.
+        // For example, the value of "name" can be set to something the player
+        // entered in a UI scene.
+        private Dictionary<string,string> PlayerInfo = new Dictionary<string, string>()
```

Comment:
The `PlayerInfo` member is sent through a signal and the type `System.Collections.Generic.Dictionary<TKey, TValue>` is not [Variant-compatible](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_variant.html#variant-compatible-types).

I assume you meant the `Godot.Collections.Dictionary<TKey, TValue>` type, if so please fully-qualify the type to avoid ambiguity for the reader.

It also needs to be changed in the delegate that declares the signal, and the method that connects to the signal.